docs/sources/tempo/metrics-generator/service_graphs/custom-service-graphs.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Custom service graphs
+menuTitle: Custom service graphs
+description: Build your own service graphs using the Node Graph visualization from Tempo generated metrics
+weight: 300
+---
+
+# Custom service graphs
+
+Depending on your need, you might want to craft custom service graphs based on the metrics generated from metrics generator, so this page is exactly about that.  
+
+## Building custom graphs
+
+
+### Prerequisites  
```

Comment:
Great that you have a prereq section! Our other doc pages use "Before you begin," with the same kind of content. I've also changed the heading level up one and removed "Building custom graphs", since the content for building custom graphs really starts in the section section. 
```suggestion
## Before you begin  
```